samples/ROPC/hosting.ini

```diff
@@ -0,0 +1,2 @@
+﻿server=Microsoft.AspNet.Server.WebListener
```

Comment:
Note: you can remove this file and set these options directly in project.json. See the Mvc sample for more info :+1: